extensions/GlobalMessages/GlobalMessagesV.i18n.php

```diff
@@ -237,6 +238,7 @@
 	'vertical-lifestyle' => 'Lifestyle vertical name',
 	'vertical-music' => 'Music vertical name',
 	'vertical-movies' => 'Movies vertical name',
+	'vertical-comedy' => 'Uncyclopedia wikis comscore tag label',
```

Comment:
I suggest to use the convention above, with a small addition -
'Comedy vertical name, used only by Uncyclopedia'